tools/harness-thci/OpenThread_BR.py

```diff
@@ -637,14 +641,14 @@ def mdns_query(self, dst='ff02::fb', service='_meshcop._udp.local', addrs_blackl
     # Override powerDown
     @API
     def powerDown(self):
-        print('powerDown')
+        self.log("Powering down BBR")
```

Comment:
I believe we should unify the criteria about what to print/log for all the `@API` decorated methods. Also notice that `'` is preferred over `"` for strings across the file.